cse-runner-app/src/main/java/com/farao_community/farao/cse/runner/app/dichotomy/RaoRunnerValidator.java

```diff
@@ -65,7 +66,7 @@ public RaoRunnerValidator(ProcessType processType,
     public DichotomyStepResult<RaoResponse> validateNetwork(Network network) throws ValidationException {
         String scaledNetworkDirPath = generateScaledNetworkDirPath(network);
         String scaledNetworkName = network.getNameOrId() + ".xiidm";
-        FileResource networkFile = fileExporter.saveNetwork(network, scaledNetworkDirPath + scaledNetworkName);
+        FileResource networkFile = fileExporter.saveNetwork(network, scaledNetworkDirPath + scaledNetworkName, GridcapaFileGroup.ARTIFACT, "", processTargetDateTime);
```

Comment:
i made some refactoring,  but still,..  we can't use the same method because in one we add a a subfolder based on the network variant name, and in the other we simply create file directly under artifacts folder of the day